processor/formatters.go

```diff
@@ -750,6 +744,24 @@ func fileSummarizeLong(input chan *FileJob) string {
 	return str.String()
 }
 
+// We need to trim the file display for tabular output formats which this does in a unicode aware way
+// to avoid cutting bytes... note that it needs to be expanded to deal with longer display characters at some
+// point in the future
+func unicodeAwareTrim(tmp string, size int) string {
+	// iterate all the runes so we can cut off correctly and get the correct length
+	var r []rune
+	for _, x := range tmp {
+		r = append(r, x)
+	}
```

Comment:
FYI, you can cast `[]rune(tmp)` the same way you would to a `[]byte`.

Reply:
Ah neat. Updated.